opengrok-indexer/src/main/java/org/opengrok/indexer/web/messages/MessagesContainer.java

```diff
@@ -227,17 +232,30 @@ public void stopExpirationTimer() {
 
         private final Instant acceptedTime = Instant.now();
 
+        // The message member is ignored so that it can be flattened using the getters
+        // specified below.
```

Comment:
I generally don't see a reason for this. Now it just duplicates/delegates all stuff from/to Message.

Now via the api you're setting different properties than you can see in the output as the input entity and output entity does not match.